packages/react-core/src/components/Menu/MenuItem.tsx

```diff
@@ -305,7 +305,7 @@ const MenuItemBase: React.FunctionComponent<MenuItemProps> = ({
             className={css(styles.menuItem, getIsSelected() && !hasCheck && styles.modifiers.selected, className)}
             aria-current={getAriaCurrent()}
             {...(!hasCheck && { disabled: isDisabled })}
-            role={hasCheck ? 'none' : 'menuitem'}
+            role={!hasCheck ? 'none' : 'menuitem'}
```

Comment:
I think this was right for the most part before - if it's not a check, it should have `role="menuitem"`,  if it's a checkbox it _shouldn't have a `role` attribute set_, which I'm assuming will just use whatever default role a `<label>` is. Sorry my previous suggestion said to set it to `role="none"` but looking at the core examples, that isn't correct.

Here are screenshots from the core examples. 

Regular non-checkbox menu
<img width="743" alt="Screen Shot 2022-05-17 at 5 35 13 PM" src="https://user-images.githubusercontent.com/35148959/168923479-65f119ff-4f78-44a4-aa13-3053a2dedf9e.png">


Checkbox menu
<img width="672" alt="Screen Shot 2022-05-17 at 5 34 47 PM" src="https://user-images.githubusercontent.com/35148959/168923456-70bb467a-a2cc-4b70-9184-918bfe7bd101.png">